.pre-commit-config.yaml

```diff
@@ -38,7 +38,7 @@ repos:
       - id: isort
         name: ISort
         description: Sort imports with isort
-        entry: isort
+        entry: poetry run isort
```

Comment:
Probably not a place to do this in this PR, but good catch

Reply:
I have about 3 other things locally that I could bundle this with. Oops.